pkgs/development/python-modules/brainflow/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildPythonPackage,
+  brainflow,
+  nptyping,
+  numpy,
+  python,
+  setuptools,
+  pythonOlder,
+}:
+
+buildPythonPackage {
+  inherit (brainflow)
+    pname
+    version
+    src
+    patches
+    ;
+
+  pyproject = true;
+  build-system = [ setuptools ];
+
+  disabled = pythonOlder "3.7";
+
+  dependencies = [
+    numpy
+    nptyping
+  ];
+
+  buildInputs = [ brainflow ];
+
+  postPatch = ''
+    cd python_package
+  '';
+
+  postInstall = ''
+    mkdir -p "$out/${python.sitePackages}/brainflow/lib/"
+    cp -Tr "${brainflow}/lib" "$out/${python.sitePackages}/brainflow/lib/"
+  '';
+
+  pythonImportsCheck = [ "brainflow" ];
+
+  meta = {
+    description = "BrainFlow is a library intended to obtain, parse and analyze EEG, EMG, ECG and other kinds of data from biosensors.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

should not mention package name and end with period.

Reply:
PING @Pandapip1

Reply:
Resolved by inheriting meta from the brainflow C library.